stage_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# The first parameter is only set locally, where we can not check the Floorist image used
+if [ -z "$1" ]; then
+    NEW_FLOORIST_IMG="$(
+        oc get deployment.apps -o jsonpath='{.items[].spec.template.spec.containers[].env[1]}{"\n"}'
+    ):$(
+        oc get deployment.apps -o jsonpath='{.items[].spec.template.spec.containers[].env[2]}{"\n"}'
```

Comment:
This takes all deployments. Can you pull it out of operator deployment, specified by its name?

This assumes the order of envs (and containers).  I think there is a possibility to match based on a `name` value.

This does not outputs the env value, but the full env object.